### PR TITLE
test(paper-form): use qunit-dom and ember-test-helpers

### PR DESCRIPTION
**Pull request by xomaczar**

### CODE REVIEW

**Comment by xomaczar** [comment]
@miguelcobain sorry did not see you merged #958, I was working on fixing `paper-form` tests

File: tests/integration/components/paper-form-test.js [hunk 1]
```diff
@@ -110,8 +110,7 @@ module('Integration | Component | paper form', function(hooks) {
       assert.notOk(isInvalidAndTouched);
     });
 
-    await click('input:first-of-type');
-    await blur('input:first-of-type');
+    await triggerEvent('input:first-of-type', 'blur');
```
Comment (miguelcobain) [inline tests/integration/components/paper-form-test.js]: Whu use `triggerEvent` instead of the `blur` helper?

Comment (xomaczar) [reply tests/integration/components/paper-form-test.js]: Using just `blur` without `click` makes this test fail - I haven’t looked at why - saw some other tests were already using `triggerEvent` - so I did the same.

**Comment by xomaczar** [comment]
Tests passed locally - need to see what’s going on with travis

**Comment by miguelcobain** [comment]
The travis error occurs randomly on progress circular tests.

**Pull request merged**